core/src/test/scala/stryker4s/config/ConfigReaderTest.scala

```diff
@@ -26,6 +31,15 @@ class ConfigReaderTest extends Stryker4sSuite with BeforeAndAfterEach {
       exc.getMessage() should include("Key not found: 'stryker4s'.")
     }
 
+    it("should fail on an unknown reporter") {
+      val confPath = FileUtil.getResource("stryker4sconfs/wrongReporter.conf")
+
+      lazy val result = ConfigReader.readConfig(confPath)
+      val exc = the[ConfigReaderException[_]] thrownBy result
+
+      exc.getMessage() should include("Cannot convert configuration")
```

Comment:
Can you also test if the ConfigReader fails on the correct line? (The one with the nonexistent reporter). Should be in the ConfigReaderFailure object off the top of my head